File: javascript/orientacaoobjetos/Cliente.js
export class Cliente {
  _nome;
  _cpf;

  get cpf(){
      return this._cpf;
  }

  get nome(){
      return this._nome;
  }

  constructor(nome, cpf) {
      this._nome = nome;
      this._cpf = cpf;
  }
}

File: kotlin/PacotesComposicao/src/main/kotlin/com/jotajr/bytebank/modelo/Cliente.kt
package com.jotajr.bytebank.modelo

class Cliente(
    val nome: String,
    val cpf: String,
    private val senha: Int
) : com.jotajr.bytebank.modelo.Autenticavel {

    override fun autentica(senha: Int): Boolean {
        if(this.senha == senha){
            return true
        }
        return false
    }

}